src/Mod/Draft/draftfunctions/svg.py

```diff
@@ -615,7 +615,7 @@ def get_svg(obj,
             svg_path += 'fill="none" '
             svg_path += 'stroke="{}" '.format(stroke)
             svg_path += 'stroke-width="{}" '.format(linewidth)
-            svg_path += 'stroke-linecap:square;'
```

Comment:
Nit: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linecap suggests that you don't need to use a CSS style property and that `stroke-linecap="square"` would've been OK as well (did not test this myself)

Reply:
Ah, good point. It still needs to be fixed, but fixing it to an attribute instead of style would probably be more consistent.

Thanks, I'll try to implement and test this later.